CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,59 @@
+### Express WG Code of Conduct
```

Comment:
This should probably be a level 1 heading; starting at level 3, then going to level 2 and back to 3 doesn't really make sense to me.